packages/front-end/components/Settings/forms/StatsEngineSelect.tsx

```diff
@@ -1,21 +1,18 @@
 import { ReactNode } from "react";
-import { UseFormReturn } from "react-hook-form";
 import { StatsEngine } from "back-end/types/stats";
-import { ProjectSettings } from "back-end/types/project";
-import { OrganizationSettings } from "back-end/types/organization";
 import { ScopedSettings } from "shared/settings";
 import SelectField from "@/components/Forms/SelectField";
 import { capitalizeFirstLetter } from "@/services/utils";
 
 export default function StatsEngineSelect({
-  form,
   parentSettings,
   showDefault = true,
   allowUndefined = true,
   label = "Statistics Engine",
+  value,
   onChange,
 }: {
-  form: UseFormReturn<OrganizationSettings | ProjectSettings>;
```

Comment:
Patterns-wise, passing the form is problematic with types. We should stick to using `onChange()` and in the handler, calling `form.setValue("statsEngine", value)` if needed.